czarjulius/Banka-React
# feat(signup): implement signup functionality-create signup action and…

#### What does this PR do?
User can register on the Banka app

#### Description of Task to be completed?
- Create signup action and types
- Create signup reducer and import in store
- Add signup functions on the signup page

#### How should this be manually tested?
- User should visit `https://banka-beact.herokuapp.com/signup` and submit his details

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#168084430](https://www.pivotaltracker.com/story/show/168084430)

#### Screenshots (if appropriate)
-N/A

#### Questions:
- N/A
